mlir/utils/performance/parameterSweeps.py

```diff
@@ -34,13 +34,30 @@ class Options:
     flags: list
     concurrent_tests: int
 
+class PerfConfig:
+    class Version(enum.Enum):
```

Comment:
Given that this is an internal utility, I think we're good to just make this be `v2` by definition?

Reply:
I can remove `V1` from `PerfConfig` class but let's leave the class. At least, it knows how to print the `perfConfig` string correctly based on a requested version. Otherwise, we will end up with lot's magic constant in the code.

Reply:
Fair enough, it's not an unreasonable restructuring

Reply:
Updated